### PR TITLE
Add new example: Actix + Rbatis

### PR DESCRIPTION
**Pull request by artileda**
Hello, Maintainer.

I added new example : Actix with Rbatis in pull request. 

Thank you.

### CODE REVIEW

File: rbatis/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#[macro_use]
+extern crate rbatis_macro_driver;
+#[macro_use]
+extern crate lazy_static;
+
+use actix_web::{get,App,HttpServer,HttpResponse,Responder,middleware::Logger};
```
Comment (jplatte) [inline rbatis/src/main.rs:6]: This example is missing spaces all over the place. Is there no CI job for `cargo fmt -- --check`?

**Comment by robjtede** [comment]
Don't worry about the nightly fail, needs a rustfmt run though.

**Review by JohnTitor — COMMENTED** [review]
The implementation looks fine to me, could you also fix the warning and run rustfmt?

File: rbatis/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#[macro_use]
+extern crate rbatis_macro_driver;
+#[macro_use]
+extern crate lazy_static;
```
[the remark this answers is not in the document]
Comment (ywnv) [reply rbatis/src/main.rs:4]: Okeyy, i'll do it.

[the remark this answers is not in the document]
Comment (ywnv) [reply rbatis/src/main.rs]: I remove rbatis_macro_driver render CRUDEnable traits missing and not compiled even i import directly from rbatis::crud::CRUDEnable, so i keep it. I still figuring it. Sorry for my ignorance.

**Comment by TDiblik** [comment]
Hi, what's a state of this pull request? Personally, I would love to see this in one of the examples

**Comment by artileda** [comment]
> Hi, what's a state of this pull request? Personally, I would love to see this in one of the examples

Stagnant in rustfmt, if you want to make one like this, i'll close this so you can do PR.

**Comment by TDiblik** [comment]
Yeah, why not, [rbatis actually provides an example](https://github.com/rbatis/rbatis/blob/master/example/src/actix_web/main.rs) on usage with actix-web (didn't know that before), so I'll just copy it, add some basic CRUD and open a new PR, is that ok?

**Comment by JohnTitor** [comment]
Wait, if rbatis has its own example code, I'd like to provide a link to it instead. Since we have a lot of example code already, they're fairly easy to be outdated, so just providing a link would reduce our work.

**Pull request closed without merging**